api/src/auth/drivers/local.ts

```diff
@@ -72,12 +74,15 @@ export function createLocalAuthRouter(): Router {
 			});
 
 			const { error } = loginSchema.validate(req.body);
-			if (error) throw new InvalidPayloadException(error.message);
+
+			if (error) {
+				throw new InvalidPayloadException(error.message);
+			}
 
 			const mode = req.body.mode || 'json';
 
 			const { accessToken, refreshToken, expires } = await authenticationService.login(
-				'local',
+				undefined, // Will default to DEFAULT_AUTH_PROVIDER
```

Comment:
Instead of defaulting to the default auth driver, this should be passing in the provider name of the current provider. That's currently not accessible in this createLocalAuthRouter() function. We should add a `providerName` parameter to the create-routes function in order for us to be able to pass it along to the authenticationsService

Reply:
Do we really have to? I assumed you hardcoded 'local' for a reason (to limit the login until the new frontend had been finalized). Previously it would use `req.params.provider` which worked fine since if it was undefined it would default to `DEFAULT_AUTH_PROVIDER`.

Either way, it had to be changed from 'local' since that isn't the default provider, which broke the login flow.